### PR TITLE
Return "point_est" when using relative stopping rule

### PR DESCRIPTION
**Pull request by yunxiao-li-psych**

### CODE REVIEW

**Comment by lucas-castillo** [comment]
Hi @yunxiao-li-psych! This looks good to me. Please update the `NEWS.md` file with the changes and I think it's ready after that.

**Review by lucas-castillo — COMMENTED** [review]
Hi @yunxiao-li-psych ! This looks good to me. Please add to tests and also update the `NEWS.md` file with the changes and I think it's ready after that.

**Comment by yunxiao-li-psych** [comment]
Hi Lucas, I have updated the NEWS.md

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# samler 1.0.2
```
Comment (lucas-castillo) [inline NEWS.md:1]: Please call it samplr 1.1.0

**Review by lucas-castillo — APPROVED** [review]

**Pull request merged**